a10_octavia/db/models.py

```diff
@@ -60,7 +60,93 @@ class VRID(base_models.BASE):
 
     id = sa.Column(sa.Integer, primary_key=True)
     project_id = sa.Column(sa.String(36), nullable=False)
-    vrid = sa.Column(sa.Integer, default=0)
+    vrid = sa.Column(sa.Integer, default=0, nullable=False)
     vrid_port_id = sa.Column(sa.String(36), nullable=False)
-    vrid_floating_ip = sa.Column(sa.String(40))
+    vrid_floating_ip = sa.Column(sa.String(40), nullable=False)
     subnet_id = sa.Column(sa.String(36), nullable=False)
+
+
+class Amphora_Meta(base_models.BASE):
+    __tablename__ = 'amphora_meta'
+
+    id = sa.Column(sa.String(36), primary_key=True, nullable=False)
+    created_at = sa.Column(u'created_at', sa.DateTime(), nullable=True)
+    updated_at = sa.Column(u'updated_at', sa.DateTime(), nullable=True)
+    last_udp_update = sa.Column(u'last_udp_update', sa.DateTime(), nullable=False)
+    status = sa.Column('status', sa.String(36), default='ACTIVE', nullable=False)
+
+
+class Thunder(base_models.BASE):
```

Comment:
The relation between thunder and thunder_cluster is not established.

Reply:
Adding thunder_cluster_id as a foreign key to thunder table to establish the relation.